specimen/contexts/archive-page.html

```diff
@@ -19,9 +19,9 @@
 
 <header>
     <div class="masthead">
-        <h1><a class="identity-logo product" href="#">Vocabulary</a></h1>
+        <h1 id="logo-heading"><a class="identity-logo product" href="#">Vocabulary</a></h1>
         <button class="expand-menu">Menu</button>
-        <nav class="primary-menu">
+        <nav class="primary-menu" aria-labelledby="logo-heading">
```

Comment:
this menu is not labelledby the logo, it instead needs to get its own arial-label

Reply:
Okay, this is noted.